2.0/.sti/bin/run

```diff
@@ -22,5 +22,5 @@ if is_puma_installed; then
   exec bundle exec "puma --config ../etc/puma.cfg"
 else
   echo "You might consider adding 'puma' into your Gemfile."
-  exec bundle exec "rackup -P ../run/rack.pid --host 0.0.0.0"
+  exec bundle exec "rackup -P ../run/rack.pid --host 0.0.0.0 --port 8080"
```

Comment:
Update port, since it's using `9292` in [here](https://github.com/openshift/sti-ruby/blob/master/2.0/test/run#L22). Or even better you can use `docker inspect` for this to be more generic and maybe that'll be a first step to making the testing more generic :wink: